config.json

```diff
@@ -1,26 +1,21 @@
 {
   "components": [
     {
-      "path": "components/app-crm-card-middle",
-      "label": "App (CRM card - middle panel)",
-      "insertPath": "./"
-    },
-    {
-      "path": "components/app-crm-card-side",
-      "label": "App (CRM card - side panel)",
+      "path": "components/example-app",
```

Comment:
Not a blocker (We can keep this as-is for now), but I wonder if we could/should try to DRY this up at some point by sharing code with what exists in the getting started project. For example, could we update the path here to be `projects/getting-started-template/src/app`?